### ISSUE TITLE
Possibility to remove weekends ?

### ISSUE THREAD

**Issue by ninoCookies** [Russian60/flex-calendar#16]
Is it possible to remove saturday and sunday from calendar ?

**Comment by VincentPuget** [comment]
No, it's currently not possible to remove weekends from calendar.

**Comment by Gabrielmtn** [comment]
Can I make them not selectable?

**Comment by Jorghe** [comment]
I created a function that checks weekdays, then store them in an array an insert it into the disabledDates, worked for me.
